flickit-assessment-core/src/main/java/org/flickit/assessment/core/application/port/in/evidence/GetEvidenceTypesUseCase.java

```diff
@@ -0,0 +1,12 @@
+package org.flickit.assessment.core.application.port.in.evidence;
+
+import java.util.List;
+
+public interface GetEvidenceTypesUseCase {
+
+    Result getEvidenceTypes();
+
+    record EvidenceTypeItem(String title, String code) {}
```

Comment:
Evidence type code is kind of an identifier and it's better to be the first parameter.

`record EvidenceTypeItem(String code, String title) {}`